modules/standard/Random.chpl

```diff
@@ -68,38 +78,48 @@ module Random {
   */
   module RandomSupport {
 
-    /*
-       An instance of :record:`SeedGenerators` that provides a
-       convenient means of generating seeds when the user does not wish to
-       specify one manually.
-    */
-    const SeedGenerator: SeedGenerators;
+    // Will be deprecated in 1.14.
+    pragma "no doc"
+    const SeedGenerator: SeedGeneratorsDeprecated;
 
+    pragma "no doc"
+    record SeedGeneratorsDeprecated {
+      proc currentTime: int(64) {
+        use Time;
+        const seed = getCurrentTime(unit=TimeUnits.microseconds):int(64);
+        const oddseed = if seed % 2 == 0 then seed + 1 else seed;
+        return oddseed;
+      }
+    }
 
```

Comment:
Support for this backwards-compatible seed generator is kind, but probably unnecessary -- I'm guessing nobody has ever used the type in a standalone format.  If we keep it for the release, I think it should at least generate a compilerWarning() in currentTime that it's deprecated and to use [whatever] instead so that those who do use it won't be surprised when it disappears.  Ideally, this deprecated feature would also introduce a new test into test/deprecated (the place that we're putting tests to remind us to pull features once 1.13 is out the door).